indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -195,6 +198,15 @@ def _process(self):
             data = {'bookings': serialized_occurrences}
         return jsonify(room_id=self.room.id, booking=reservation_details_schema.dump(resv), calendar_data=data)
 
+    @staticmethod
+    @make_interceptable
+    def after_create(args: dict[str, typing.Any], reservation: 'Reservation') -> None:
+        """
+        Hook called after a reservation is created.
+        :param args: The data from the request that was used to create the Reservation.
+        :param reservation: The Reservation object that was created.
+        """
```

Comment:
```suggestion
        """Hook called after a reservation is created.

        :param args: The data from the request that was used to create the Reservation.
        :param reservation: The Reservation object that was created.
        """
```

Reply:
Updated in https://github.com/indico/indico/pull/6126/commits/07b7a5b4c8cfa60b78a1729483d63164aae66801, thanks!